src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,72 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] stack;
+    private static final int SIZE = 10;
```

Comment:
This is better bus still not good name. Firstly it's repeated the same name of changeable variable. Second nothing in this name shows that it's constant value. Use some helper word to show that this value is constant (and this word shouldn't be "constant"). To find example of such a names look inside some java core classes such as ArrayList or HashMap and compare there names with your